README: correction, improvements (#4)
ifreund/pkgbasify: README.md

It is highly recommended to make backups before running this tool.

That said, I am not aware of any bugs in pkgbasify, and have used it to successfully upgrade test systems.
If you find a bug in pkgbasify, please open an issue.

## Behavior

pkgbasify performs the following steps:

1. Make a copy of the [etcupdate(8)](https://man.freebsd.org/cgi/man.cgi?query=etcupdate&sektion=8&manpath=freebsd-release) current database (`/var/db/etcupdate/current`) so that pkgbasify can merge config files after converting the system.
2. Select a repository based on the output of [freebsd-version(1)](https://man.freebsd.org/cgi/man.cgi?query=freebsd-version&sektion=1&manpath=freebsd-release) and create `/usr/local/etc/pkg/repos/FreeBSD-base.conf`.
3. Select packages that correspond to the currently installed base system components.
   - For example: if the lib32component is not already installed, pkgbasify will skip installation of lib32 packages.
5. Install the selected packages with [pkg(8)](https://man.freebsd.org/cgi/man.cgi?query=pkg&sektion=8&manpath=freebsd-ports), overwriting base system files, and creating `.pkgsave` files as per standard `pkg(8)` behavior.

> 6. Run a three-way-merge between the `.pkgsave` files (ours), the new files installed by pkg (theirs), and the old files in the copy of the etcupdate database.

Author: ```suggestion
6. Run a three-way-merge: the copy of the etcupdate database, `.pkgsave` files, and pkgbase-provided files.
```

Trying to avoid _ours_, _theirs_, _old_, and _new_ … I might revisit this.

Reviewer: I think this change reduces clarity for anyone who understands how a 3-way-merge works.
